File: test_socket.py
import socketio
import time

sio = socketio.Client()

@sio.event
def connect():
    print('connected to server')

@sio.event
def packet(data):
    print('received packet:', data)

sio.connect('http://localhost:3001')
time.sleep(3)
sio.disconnect()

File: test_socket.js
const io = require('socket.io-client');
const socket = io('http://localhost:3001');
socket.on('connect', () => {
  console.log('Connected!');
});
socket.on('packet', (data) => {
  console.log('Packet:', data);
});
setTimeout(() => {
  socket.disconnect();
}, 2000);
